File: ce/customer-engagement/on-premises/snippets/csharp/CRMV8/query/cs/linqexamples49.cs
// <snippetlinqexamples49>


 using (ServiceContext svcContext = new ServiceContext(_serviceProxy))
 {
  var query_multiselect = svcContext.IncidentSet
                         .Where(i => i.IncidentId != _incidentId1)
                         .Select(i => i.incident_customer_accounts)
                         .Where(a => a.AccountId != _accountId2)
                         .Select(a => a.account_primary_contact)
                         .OrderBy(c => c.FirstName)
                         .Select(c => c.ContactId);
  foreach (var c in query_multiselect)
  {
   System.Console.WriteLine(c.GetValueOrDefault());
  }
 }

// </snippetlinqexamples49>

File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/queryvb/vb/linqexamples49.vb
' <snippetlinqexamples49>


 Using svcContext As New ServiceContext(_serviceProxy)
  Dim query_multiselect =
      svcContext.IncidentSet _
      .Where(Function(i) Not i.IncidentId.Value.Equals(_incidentId1)) _
      .Select(Function(i) i.incident_customer_accounts) _
      .Where(Function(a) Not a.AccountId.Value.Equals(_accountId2)) _
      .Select(Function(a) a.account_primary_contact) _
      .OrderBy(Function(c) c.FirstName).Select(Function(c) c.ContactId)
  For Each c In query_multiselect
   Console.WriteLine(c.GetValueOrDefault())
  Next c
 End Using

' </snippetlinqexamples49>